test/new-e2e/tests/agent-shared-components/forwarder/nss_failover_test.go

```diff
@@ -18,33 +18,47 @@ import (
 	"text/template"
 	"time"
 
-	"github.com/DataDog/test-infra-definitions/components/datadog/agent"
-	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
-	awsResources "github.com/DataDog/test-infra-definitions/resources/aws"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/fakeintake/fakeintakeparams"
-	"github.com/DataDog/test-infra-definitions/scenarios/aws/vm/ec2vm"
 	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 
+	"github.com/DataDog/test-infra-definitions/components/datadog/agent"
+	"github.com/DataDog/test-infra-definitions/components/datadog/agentparams"
+	"github.com/DataDog/test-infra-definitions/resources/aws"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/ec2"
+	"github.com/DataDog/test-infra-definitions/scenarios/aws/fakeintake"
+
 	fi "github.com/DataDog/datadog-agent/test/fakeintake/client"
-	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/components"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/e2e"
 	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client"
+	"github.com/DataDog/datadog-agent/test/new-e2e/pkg/utils/e2e/client/agentclient"
 )
 
 type multiFakeIntakeEnv struct {
-	VM          client.VM
-	Agent       client.Agent
-	Fakeintake1 *client.Fakeintake
-	Fakeintake2 *client.Fakeintake
+	Host        *components.RemoteHost
+	Agent       *components.RemoteHostAgent
+	Fakeintake1 *components.FakeIntake
+	Fakeintake2 *components.FakeIntake
+}
+
+func (e *multiFakeIntakeEnv) Init(ctx e2e.Context) error {
+	if e.Agent != nil {
+		agent, err := client.NewHostAgentClient(ctx.T(), e.Host, true)
+		if err != nil {
+			return err
+		}
+		e.Agent.Client = agent
+	}
+
+	return nil
 }
 
 const (
 	// intakeName should only contains alphanumerical characters to ease pattern matching /etc/hosts
 	intakeName      = "ddintake"
-	fakeintake1Name = "1fakeintake"
-	fakeintake2Name = "2fakeintake"
+	fakeintake1Name = "1"
+	fakeintake2Name = "2"
```

Comment:
Any particular reason for the renaming ? Having a distinctive name was very convenient when reading the logs of the test...

Reply:
The component now adds the `fakeintake` [part itself](https://github.com/DataDog/test-infra-definitions/blob/b181ca52249a54e12931386f368539883e435ec2/scenarios/aws/fakeintake/fakeintake.go#L46-L47), so removed here to avoid duplication;